packages/shared/src/variants/freeze.ts

```diff
@@ -32,6 +40,34 @@ export class FreezeGo extends Baduk {
   override exportState(): FreezeGoState {
     return { ...super.exportState(), frozen: this.frozen };
   }
+
+  static uiTransform(config: NewBadukConfig, state: FreezeGoState) {
```

Comment:
Note: we've added ~40 lines to this file, but removed over 100 lines by deleteing FreezeBoard.vue!

This is the benefit I am looking for (in addition to making boards more consistent)